models/redirect/fields.yaml

```diff
@@ -114,6 +114,12 @@ tabs:
             span: left
             default: false
             comment: winter.redirect::lang.redirect.ignore_trailing_slash_comment
+        forward_query_parameters:
+            label: winter.redirect::lang.redirect.forward_query_parameters
+            type: checkbox
+            span: left
+            default: false
```

Comment:
I've made the other options default to enabled, should we do the same here?

Reply:
Added in a90bea790141102a2e655e650f3faeb99f6703fd